response/core/serializers.py

```diff
@@ -10,6 +10,12 @@ class Meta:
         fields = ("app_id", "external_id", "display_name")
 
 
+class TimelineEventSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = TimelineEvent
+        fields = ("timestamp", "text", "event_type", "metadata")
```

Comment:
Not a blocker, but will metadata be mapped back to a string or will it be a tuple index?

Reply:
Ah, I see it's stored as a text field 👍

Reply:
It _should_ be serialized as nested JSON in the API response.

In the model it corresponds to a JSONField, which is only supported in Postgres - so this is tricky to automatically test